### DISCUSSION TITLE
Add support for Relay Cursor Connections Spec (relay.dev)

### DISCUSSION THREAD

**Discussion by taraktikos** [refinedev/refine#3472] [Integration request]
### Is your feature request related to a problem? Please describe.

Relay is designed for high performance at any scale. Relay keeps management of data-fetching easy, whether your app has tens, hundreds, or thousands of components. And thanks to Relay’s incremental compiler, it keeps your iteration speed fast even as your app grows 

### Describe the thing to improve

Create a Dataprovider that supports communication with backends that implement the [Relay Cursor Connections Spec](https://relay.dev/graphql/connections.htm)

**Comment by omeraplak** [comment]
Hey @taraktikos ,
Unfortunately, we do not have enough experience and knowledge about Relay to make this implementation. I'm sure the community will help if technically possible 🚀
